train-A-app/src/app/auth/components/signup/signup.component.scss

```diff
@@ -0,0 +1,15 @@
+@import '@styles/function.scss';
+@import '@styles/variables.scss';
+@import '@styles/common.scss';
+
+.signup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $primary;
+  height: 95vh;
+
+  &__form {
```

Comment:
I add this code and responsive width to common file. I think you should remove it

Reply:
background-color became white

Reply:
Yep, I've changed it